tools/src/icon4py/tools/py2fgen/settings.py

```diff
@@ -127,17 +127,12 @@ def device(self) -> Device:
         device = device_map[self.icon4py_backend]
         return device
 
-    @cached_property
-    def limited_area(self) -> bool:
-        return env_flag_to_bool("ICON4PY_LAM", False)
-
     @cached_property
     def parallel_run(self) -> bool:
-        return env_flag_to_bool("ICON4PY_PARALLEL", False)
```

Comment:
The default should be to use the MPI exchange. I kept the flag for debugging for now.